head/head.go

```diff
@@ -111,6 +107,17 @@ func NewHead(ctx context.Context, options ...opts.Option) (*Head, chan Bootstrap
 			providers.Cache(cache),
 		}))
 	}
+	if cfg.DisableValues {
+		dhtOpts = append(dhtOpts, dht.DisableValues())
```

Comment:
This is good. Right now the DHT still has the validators attached even when values are disabled which is a bit of a bug, but we should fix that in the DHT and I don't think it should cause any problems here.